gating/thaw/run

```diff
@@ -34,7 +34,7 @@ export ANSIBLE_STDOUT_CALLBACK=debug
 # IP fact being up to date.
 export ANSIBLE_GATHERING=implicit
 openstack-ansible -v /opt/rpc-openstack/gating/thaw/thaw.yml
-openstack-ansible -t haproxy_server-config haproxy-install.yml
+openstack-ansible -t haproxy-server haproxy-install.yml
```

Comment:
Perhaps it's better just to remove the tag and let the whole role run? That way if tags are changed again, it doesn't matter.